### ISSUE TITLE
[Bug] V3 Hyperparameter Tuning Pipeline page labelled "Download Data" in navigation due to missing title cell

### ISSUE THREAD

**Issue by admivsn** [aws/sagemaker-python-sdk#6232]
**PySDK Version**
- [ ] PySDK V2 (2.x)
- [ x] PySDK V3 (3.x)

**Describe the bug**
The [V3 Hyperparameter Tuning Pipeline example](https://sagemaker.readthedocs.io/en/stable/v3-examples/ml-ops-examples/v3-hyperparameter-tuning-example/v3-hyperparameter-tuning-pipeline.html) notebook is missing a title markdown cell. The notebook's first cell is a code cell (imports), so the first markdown heading `### Download Data`is picked up as the page label in the readthedocs navigation sidebar, causing the page to appear as "Download Data" rather than "V3 Hyperparameter Tuning Pipeline".

**To reproduce**
1. Navigate to the [readthedocs page](https://sagemaker.readthedocs.io/en/stable/v3-examples/ml-ops-examples/v3-hyperparameter-tuning-example/v3-hyperparameter-tuning-pipeline.html)
2. Observe the sidebar navigation label for this page

**Expected behavior**
The page should be labelled "V3 Hyperparameter Tuning Pipeline" (or similar) in the navigation sidebar, not "Download Data".

**Screenshots or logs**
<img width="1169" height="919" alt="Image" src="https://github.com/user-attachments/assets/bb1dc714-cb88-42e3-85fd-4637ce676a4b" />

**System information**
A description of your system. Please provide:
- **SageMaker Python SDK version**:
- **Framework name (eg. PyTorch) or algorithm (eg. KMeans)**:
- **Framework version**:
- **Python version**:
- **CPU or GPU**:
- **Custom Docker image (Y/N)**:

**Additional context**
Fix: add a title markdown cell as the first cell in doc/source/v3-examples/ml-ops-examples/v3-hyperparameter-tuning-example/v3-hyperparameter-tuning-pipeline.ipynb:

```
{
  "cell_type": "markdown",
  "id": "intro",
  "metadata": {},
  "source": ["# V3 Hyperparameter Tuning Pipeline"]
}
```
